tests/main/views/test_direct_award.py

```diff
@@ -445,7 +445,7 @@ def test_end_search_page_renders_error_when_results_more_than_limit(self):
             "services": [],
             "meta": {
                 "query": {},
-                "total": 1000,
+                "total": 31,
```

Comment:
How about we change this to use the setting instead? 

```
with self.app.app_context():
    self._search_api_client_helpers._get.return_value = {
        "services": [],
        "meta": {
            "query": {},
            "total": current_app.config['END_SEARCH_LIMIT'] + 1,
            "took": 3
        },
        "links": {}
    }
```

Reply:
Ah wait - it's not in the config is it 😹 Even easier then - just import from `app.views.g_cloud.END_SEARCH_LIMIT`.

Reply:
good idea 👍